Makefile

```diff
@@ -0,0 +1,14 @@
+help:	## Display this message
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+.PHONY: help
+.DEFAULT_GOAL := help
+
+test: ## Run tests
+	go test -race -coverprofile=coverage.txt -covermode=atomic ./...
+.PHONY: test
+
+lint: ## Run linters
```

Comment:
Currently, this target might not be very useful from a CI perspective.

`go fmt ./...` runs `gofmt -w -l` behind the scenes which basically overrides the contents of the files (`-w`) and then exits with `0`, therefore, the CI will not fail. In a similar note, `go run golang.org/x/lint/golint` will not exit with an error code either. 

If the idea is to be more strict on this target these are a couple of suggestions (I'm not sure if there are better ways of doing this):
```
// go fmt ./...
test -z $(shell gofmt -l ./..) || (gofmt -l -d ./..; exit 1)

// go run golang.org/x/lint/golint ./...
go run golang.org/x/lint/golint -set_exit_status ./...
```

Feel free to ignore this comment if this target is not meant to fail in case of suggested changes.